client/blocks/simple-site-rename-form/index.jsx

```diff
@@ -0,0 +1,107 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { noop, get, flow } from 'lodash';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import { getPrimaryDomainBySiteId, getPrimarySiteId } from 'state/selectors';
+import Card from 'components/card';
+import CompactCard from 'components/card/compact';
+import FormButton from 'components/forms/form-button';
+import FormTextInputWithAffixes from 'components/forms/form-text-input-with-affixes';
+import ConfirmationDialog from './dialog';
+
+export class SimpleSiteRenameForm extends Component {
+	state = {
+		showDialog: false,
+		domainFieldValue: '',
+	};
+
+	onSubmit = event => {
+		event.preventDefault();
+
+		this.setState( {
+			showDialog: true,
+		} );
+	};
+
+	onDialogClose = () => {
+		this.setState( {
+			showDialog: false,
+		} );
+	};
+
+	onFieldChange = event => {
+		// Debounce?
+		this.setState( { domainFieldValue: get( event, 'target.value' ) } );
+	};
+
+	render() {
+		const { primaryDomain, translate } = this.props;
+
+		// This may need more consideration given the new x.music.blog type domains
+		const isWPCOM = primaryDomain.type === 'WPCOM';
+		// This is naive - there's surely a better way.
+		const domainPrefix = primaryDomain.name.replace( '.wordpress.com', '' );
+
+		const isDisabled =
+			! isWPCOM || ! this.state.domainFieldValue || this.state.domainFieldValue === domainPrefix;
+
+		return (
+			<div className="simple-site-rename-form">
+				<ConfirmationDialog
+					isVisible={ this.state.showDialog }
+					onClose={ this.onDialogClose }
+					newDomainName={ this.state.domainFieldValue }
+					currentDomainName={ primaryDomain.name.replace( '.wordpress.com', '' ) }
+				/>
+				<form onSubmit={ this.onSubmit }>
+					<Card className="simple-site-rename-form__content">
+						<FormTextInputWithAffixes
+							type="text"
+							value={ this.state.domainFieldValue }
+							suffix={ '.wordpress.com' }
+							onBlur={ noop }
+							onChange={ this.onFieldChange }
+							onFocus={ noop }
+							placeholder={ domainPrefix }
+						/>
+						<div className="simple-site-rename-form__info">
+							<Gridicon icon="info-outline" size={ 18 } />
+							<p>
+								Some notes about domains <a>Learn More</a>
+							</p>
+						</div>
+					</Card>
+					<CompactCard className="simple-site-rename-form__footer">
+						<FormButton disabled={ isDisabled } type="submit">
+							{ translate( 'Change Site address', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **39** times:
`translate( 'Change Site Address' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/simple-sites-rename-form)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.